docs/rules/prefer-const.md

```diff
@@ -57,18 +79,75 @@ for (let i = 0, end = 10; i < end; ++i) {
     console.log(a);
 }
 
-// the initializer is located at another block.
-let a;
-if (true) {
-    a = 0;
-}
-console.log(a);
-
 // suggest to use `no-var` rule.
 var b = 3;
 console.log(b);
 ```
 
+## Options
+
+```json
+{
+    "prefer-const": ["error", {"destructuring": "any"}]
+}
+```
+
+### destructuring
+
+The kind of the way to address variables in destructuring.
+There are 2 values:
+
+* `"any"` (default) - If any variables in destructuring should be `const`, this rule warns the variables.
```

Comment:
I think this should be "...warn the variable."  It only warns for the variable that should be `const` and not all the destructured variables, correct?